### ISSUE TITLE
Changes to config not being used by plugin

### ISSUE THREAD

**Issue by nolani12** [NoCheatPlus/Issues#214]
It's like the plugin is ignoring the config. I make changes and the plugin does not honor them at all - prefix, checks, strings, all parts of the config.

I'm on 3.15.0-SNAPSHOT-sMD5NET-b1015.

Please look into this, players can't even use elytra (which will be a rank perk) because NCP is blocking them from doing so, and it is not honoring the changes I save in the config.

Full `/ncp version` output:

`22.06 12:32:11 [Server] INFO ---- Version information ----`

`22.06 12:32:11 [Server] INFO #### Server ####`

`22.06 12:32:11 [Server] INFO git-Spigot-3ccadba-fb568fd ~MC: 1.10~`

`22.06 12:32:11 [Server] INFO detected: 1.10`

`22.06 12:32:11 [Server] INFO #### NoCheatPlus ####`

`22.06 12:32:11 [Server] INFO Plugin: 3.15.0-SNAPSHOT-sMD5NET-b1015`

`22.06 12:32:11 [Server] INFO MCAccess: 1.10 / Spigot-CB-1.10_R1`

`22.06 12:32:11 [Server] INFO Features:`

`22.06 12:32:11 [Server] INFO blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 |`

`BlocksMC1_7_2 | BlocksMC1_8 | BlocksMC1_9 | BlocksMC1_10`

`22.06 12:32:11 [Server] INFO checks: FastConsume | Gutenberg | HotFixFallingBlockPortalEnter`

`22.06 12:32:11 [Server] INFO defaults: pvpKnockBackVelocity`

`22.06 12:32:11 [Server] INFO Hooks: AllViolations~NCP~ 1.0`

`22.06 12:32:11 [Server] INFO #### Related Plugins ####`

`22.06 12:32:11 [Server] INFO ProtocolLib v4.0.1`

**Comment by MarkElf** [comment]
I may as well ask for following additional information:
-Do your start up logs show any errors upon startup? If you made any mistakes they will appear there; can you provide the startup log? (/log/latest.txt in your main directory - folder with world files)
-Is it possible that you have entered a value that NCP does not accept? If you are willing to, please upload it to pastebin under unlisted.
-How are you attempting to load the configuration? Try any of the following:
 ->Editing the configuration while the server is offline, Ensure that you hit **control (ctrl) + s** on your keyboard upon closing the configuration to ensure that it saves.
 ->Edit the configuration while the server is online and run "/ncp reload"
 ->never use /reload as 9 out of 10 plugins are just going to crash and not boot up properly.

Hopefully I contributed somewhat towards resolving your problem as it's hard to identify the source of the issue with the limited presented information.

**Comment by nolani12** [comment]
Thanks for replying!

As far as that goes...

- There are no errors during startup. I just restarted the server and watched the console. No NCP related errors.
- Here is my config file: http://pastebin.com/KjZYLUqR - I even deleted the old one and freshly generated it, changed one setting (the elytra check value) and it still didn't like it.
- When I edit the configuration, I always make sure it saves. My ftp client (transmit) always tells me with a notification that the file is uploaded. Then I do /ncp reload. I never use /reload, this normally will crash my server, lol. 
- I have updated ProtocolLib, thanks.
- I'm using the latest BuildTools Spigot.

The issue is quite vague, and therefore hard to describe. The plugin basically flat out ignores the changes I made in the config :( and I don't know why. I'm using the latest NCP build as well. Is there anything else I can provide?
